source/settings/core.rst

```diff
@@ -2868,6 +2868,20 @@ See :ref:`settings` for list of all setting groups.
    As used here, the variable ``%d`` expands to the domain of the local user.
    Other :ref:`mail user variables <variables-mail_user>` can be used as well.
 
+.. dovecot_core:setting:: process_shutdown_filter
+   :values: @string
+
+   .. versionadded:: 2.3.18
+
+   Filter to specify which events stop new connections to the process.  The syntax
+   of the filter is described in :ref:`event_filter_global`.
```

Comment:
a bit clearer: `Filter to specify which events shutdown the process after finishing the current connections.`

Also append: `This is mainly intended to save memory by preventing long-running imap processes that use a lot of memory (due to libc not freeing all of it to the OS).`